OrbitGl/OpenGlDetect.cpp

```diff
@@ -0,0 +1,68 @@
+#include "OpenGlDetect.h"
+
+#include <absl/strings/str_split.h>
+
+#include <charconv>
+
+#include "OpenGl.h"
+#include "OrbitBase/Logging.h"
+
+namespace OrbitGl {
+
+std::optional<OpenGlVersion> DetectOpenGlVersion(GlutContext*) {
+  const auto window =
```

Comment:
I wonder if "glutHideWindow()" could prevent the flickering.

Reply:
This didn't work for me when I tried it. But it also took a long time to figure out that I have to call glutMainLoopEvent several times. Maybe this has to be combined properly.